### ISSUE TITLE
add docker-compose & many workers remark to readme

### ISSUE THREAD

**Issue by rriemann** [Elico-Corp/odoo-docker#38]
Hey Sébastien (ça me semble français),

I'm currently investigating how I can install in the most reproducable way a fast responding odoo on a small server (2 cores, 4GB, SSD) for an NGO with few money. As migration from 8.0 to 9.0 is a complicated thing, I thougt about going with 9.0 and also tried your template.

It woud be cool if you could include an example `docker-compose.yml` in the readme.
I would be interested as well if your image supports settings with workers > 0. The official docker odoo/odoo:9.0 is missing some dependencies odoo/docker#42 .

This is my take on a docker-compose.yml:

``` yml
db:
  container_name: odoo-db-server
  environment:
    POSTGRES_USER: odoo
    POSTGRES_PASSWORD: odoo
  image: postgres:9.4
  volumes_from:
    - db-data

# PostgreSQL data files
db-data:
  container_name: odoo-db-data
  image: postgres:9.4
  command: "true"

# Odoo server
app:
  container_name: odoo-app-server
  ports:
    - 0.0.0.0:80:8069/tcp
  restart: always
  image: elicocorp/odoo:9.0
  links:
    - db:db
#  volumes:
#     - ./etc:/opt/odoo/etc
#     - ./addons:/opt/odoo/additional_addons:ro
#  volumes_from:
#    - app-data

# app-data:
#   container_name: odoo-app-data
#   image: elicocorp/odoo:9.0
#   command: "true"
```

**Comment by seb-elico** [comment]
Hi Robert,
Indeed I'm french ;)
Thanks for the suggestion! Actually this Docker image is still a work in progress (the current version is almost exactly the same as XCG image), we're working on an enhanced version with:
- automatic pull of additional_addons from a GitHub project
- automatic pull of OCA, community, etc. modules using oca_dependencies.txt
- setup of odoo.conf parameters through environment variables (currently you need to build a local image in which you replace odoo.conf, like in the example below)
```
FROM elicocorp/odoo:9.0
User 0
ADD odoo.conf /opt/odoo/etc/odoo.conf
```
Until we have something stable, I didn't bothered creating any documentation but hopefully I will :)

**Comment by rriemann** [comment]
Thanks for your reply. I would suggest to work with host volumes ./etc to control the config from outside of the container. it doesn't make so much sense to me to create an environment variable for every odoo option…

Unfortunately, the xcgd image has only an out-dated 9.0 alpha. :(

**Comment by elicoidal** [comment]
Closing for no activity in many months. Open again if necessary :)

**Comment by rriemann** [comment]
No problem. I abandoned odoo long ago, because I had the impression that the enterprise behind odoo is not interested in a frictionless deployment without paying for them. I hope things have changed in the meanwhile.

**Comment by seb-elico** [comment]
Fixed in #70
